### DISCUSSION TITLE
cinema mod doesn't work

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#16997] [Q&A] [answered]
it just doesn't work

https://user-images.githubusercontent.com/78441708/155857759-bae08e2e-d9bc-41dc-9fd5-176a6e363bcb.mp4

[logs.zip](https://github.com/ppy/osu/files/8147426/logs.zip)

**Comment by frenzibyte** [comment]
Please attach your beatmap. It's working perfectly fine on a couple of beatmaps I have locally.

**Comment by bdach** [comment]
Note that you have storyboards turned off. Does bring up a point as to whether they should automatically be toggled on, if that is what you're expecting to see (hard to tell without knowing what the map is).

**Reply by frenzibyte** [reply]
I've noticed that at first, but it turns out "Cinema" already has its own logic of ignoring user settings (which includes the "Storyboard / Video" checkbox).

**Reply by bdach** [reply]
Yep, good spot, fair enough

**Comment by iSimpFor** [comment]
i feel like this is related to #16959

**Comment by Axilotl17** [comment, marked as answer]
sorry, im just now realizing you aren't supposed to see any notes; i was under the impression it was autoplay but without controls or timeline, not sure why i thought that.

**Reply by uzervlad** [reply]
aren't notes kinda visible with cinema in stable? is it not the case with lazer?

**Reply by frenzibyte** [reply]
They're barely ever noticeable, but the idea remains that "Cinema" mod is for hiding the playfield completely and only view the storyboard with the hitsounds playing in the background.
